swagger/docs/swagger.json

```diff
@@ -479,6 +513,13 @@
                 "owner": {"required": true, "type": "string"}
             }
         },
+        "UpdatedCarpool": {
```

Comment:
That will come later. I wanted to get this figured out first. I'll start another branch for that.